docs/examples/s2n_negotiate_example.c

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Does the file need `_example.c` in it? Doesn't being in the `examples` directory already imply that?